internal/app/vodtester/vodtester_app.go

```diff
@@ -0,0 +1,161 @@
+package vodtester
+
+import (
+	"context"
+	"time"
+
+	"github.com/golang/glog"
+	api "github.com/livepeer/go-api-client"
+)
+
+type (
+	// IVodTester ...
+	IVodTester interface {
+		// Start start test. Blocks until finished.
+		Start(fileName string, testDuration, pauseDuration time.Duration) (int, error)
+		Cancel()
+		Done() <-chan struct{}
+		Clean()
+	}
+
+	VodTesterOptions struct {
+		API *api.Client
+	}
+
+	vodTester struct {
+		ctx    context.Context
+		cancel context.CancelFunc
+		lapi   *api.Client
+	}
+)
+
+// NewVodTester ...
+func NewVodTester(gctx context.Context, opts VodTesterOptions) IVodTester {
+	ctx, cancel := context.WithCancel(gctx)
+	vt := &vodTester{
+		lapi:   opts.API,
+		ctx:    ctx,
+		cancel: cancel,
+	}
+	return vt
+}
+
+func (vt *vodTester) Start(fileUrl string, testDuration, pauseDuration time.Duration) (int, error) {
```

Comment:
Both `testDuration` and `pauseDuration` are actually unused in `vodTester`. Should remove the args.